test/aeso_abi_tests.erl

```diff
@@ -5,27 +5,20 @@
 
 -define(SANDBOX(Code), sandbox(fun() -> Code end)).
 -define(DUMMY_HASH_WORD, 16#123).
--define(DUMMY_HASH, <<0:30/unit:8, 127, 119>>). %% 16#123
 -define(DUMMY_HASH_LIT, "#0000000000000000000000000000000000000000000000000000000000000123").
 
 sandbox(Code) ->
     Parent = self(),
     Tag    = make_ref(),
     {Pid, Ref} = spawn_monitor(fun() -> Parent ! {Tag, Code()} end),
     receive
-        {Tag, Res} -> erlang:demonitor(Ref, [flush]), {ok, Res};
-        {'DOWN', Ref, process, Pid, Reason} -> {error, Reason}
+        {Tag1, Res} when Tag1 =:= Tag -> erlang:demonitor(Ref, [flush]), {ok, Res};
```

Comment:
🙈 Nothing wrong with the original code, this is neither shorter nor more readable?